universal-application-tool-0.0.1/test/controllers/admin/AdminQuestionControllerTest.java

```diff
@@ -245,6 +248,41 @@ public void update_redirectsOnSuccess() {
     assertThat(result.flash().get("message").get()).contains("updated");
   }
 
+  @Test
+  public void update_setsIdsAsExpected() {
+    DropdownQuestionDefinition definition =
+        new DropdownQuestionDefinition(
+            "applicant ice cream",
+            Optional.empty(),
+            "Select your favorite ice cream flavor",
+            LocalizedStrings.of(Locale.US, "Ice cream?", Locale.FRENCH, "crème glacée?"),
+            LocalizedStrings.of(Locale.US, "help", Locale.FRENCH, "aider"),
+            ImmutableList.of(
+                QuestionOption.create(
+                    1L, LocalizedStrings.of(Locale.US, "chocolate", Locale.FRENCH, "chocolat")),
+                QuestionOption.create(
+                    2L, LocalizedStrings.of(Locale.US, "strawberry", Locale.FRENCH, "fraise")),
+                QuestionOption.create(
+                    3L, LocalizedStrings.of(Locale.US, "vanilla", Locale.FRENCH, "vanille")),
+                QuestionOption.create(
+                    4L, LocalizedStrings.of(Locale.US, "coffee", Locale.FRENCH, "café"))));
+    // We can only update draft questions, so save this in the DRAFT version.
+    testQuestionBank.maybeSave(definition, LifecycleStage.DRAFT);
+
+    ArrayList<String> newOptions = new ArrayList<>();
+    newOptions.add("cookie");
+    newOptions.add("mint");
+    newOptions.add("pistachio");
+
+    DropdownQuestionForm questionForm = new DropdownQuestionForm(definition);
+    questionForm.setNewOptions(newOptions);
```

Comment:
suggestion: You can inline and remove the one-off array with ImmutableList.of("cookie", "mint"...)'